client/src/ts/dd_main.ts

```diff
@@ -3,6 +3,17 @@ import dial = require('./dial');
 import readout = require('./readout');
 import vis = require('./visibility');
 import animate = require('./animate');
+import canDefs = require('./can_msg_defs');
+
+// Helpers
+function toFixedTrunc(value: number, n: number): string {
```

Comment:
The fact that this is necessary bothers me. You can't use `.toPrecision()`?

Reply:
Done.